src/objects.js

```diff
@@ -5,34 +5,94 @@ const keys = (obj) => {
   // Retrieve all the names of the object's properties.
   // Return the keys as strings in an array.
   // Based on http://underscorejs.org/#keys
+  return Object.keys(obj);
 };
 
 const values = (obj) => {
   // Return all of the values of the object's own properties.
   // Ignore functions
-  // http://underscorejs.org/#values
+  // http://underscorejs.org/#
+
+  const objKeys = Object.keys(obj);
+  const result = [];
+
+  objKeys.forEach((k) => {
+    return result.push(obj[k]);
+  });
+
+  // for (const key in obj) {
```

Comment:
You can just deleted code that you've commented out. It makes for a cleaner submission.